tcmu-runner.service

```diff
@@ -4,5 +4,5 @@ Description=LIO Userspace-passthrough daemon
 [Service]
 Type=dbus
 BusName=org.kernel.TCMUService1
+KillMode=process
```

Comment:
We run this in cloud environments, we need it to recover upon failure with no human interaction.  The failure should be logged in syslog, though.